### DISCUSSION TITLE
"The question about Nuclei INTERACTSH options?"

### DISCUSSION THREAD

**Discussion by charis3306** [projectdiscovery/nuclei#5108] [Q&A]
Can nuclei perform custom dnslog? Why don't I have the -iserver option when running nuclei?
"Help, how should I use it?"
![image](https://github.com/projectdiscovery/nuclei/assets/84823804/c3c279d5-2779-47a3-984b-60078d66f941)

**Comment by geeknik** [comment]
`-iserver your_interactsh_server_name_or_ip` plus `-itoken` and the client token you see in the interactsh-server output. 👍🏻

**Reply by charis3306** [reply]
The first step
![image](https://github.com/projectdiscovery/nuclei/assets/84823804/09a56fa8-36f6-40f3-a49a-bd2c17772a06)
The second step
![image](https://github.com/projectdiscovery/nuclei/assets/84823804/dadc653d-8a74-40ed-bd2d-00d509fcdc5e)
The third step
![image](https://github.com/projectdiscovery/nuclei/assets/84823804/83d1fd1d-cc57-42be-af25-37b668a04e93)
run
![image](https://github.com/projectdiscovery/nuclei/assets/84823804/b413a263-23fc-4f50-a1be-edcc2ad1a46f)
"I solved this problem."

**Reply by charis3306** [reply]
Thank you.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
